monitoring/grafana/dashboards/get_into_teaching_api.json

```diff
@@ -1680,10 +1692,10 @@
           "refId": "A"
         },
         {
-          "expr": "sum(increase(http_requests_received_total{controller=~\".+\",action=~\".+\",code=~\"4..\"}[$__range]))",
+          "expr": "sum(increase(http_requests_received_total{controller=~\".+\",action=~\".+\",code=~\"4.[^4]\"}[$__range]))",
```

Comment:
Will this also omit `414` (Request-URI Too Long) and `424` (Failed Dependency) errors?